bitswap/internal/defaults/defaults.go

```diff
@@ -34,3 +36,9 @@ const (
 	MaximumHashLength = 128
 	MaximumAllowedCid = binary.MaxVarintLen64*4 + MaximumHashLength
 )
+
+var (
+	// RebroadcastDelay is the default delay to trigger broadcast of
+	// random CIDs in the wantlist.
+	RebroadcastDelay = delay.Fixed(time.Minute)
```

Comment:
Does this needs to be a var ? I would just use a `time.Duration` if `go-ipfs-delay` can't be constified.

Reply:
fair enough, I had also forgotten to actually use this value. Fixed in latest commit.